server/jetstream_cluster.go

```diff
@@ -2050,7 +2050,15 @@ func (js *jetStream) createRaftGroup(accName string, rg *raftGroup, storage Stor
 	}
 
 	// Check if we already have this assigned.
+retry:
 	if node := s.lookupRaftNode(rg.Name); node != nil {
+		if node.State() == Closed {
```

Comment:
Feels like there may be a better approach here.

Also I think we need to check that the peer lists are the same if we are going to re-use.

Reply:
Do we want to return a closed node here?